### PR TITLE
Don't create duplicate VM session in a same VMCheck instance

### PR DESCRIPTION
**Pull request by xiaodwan**
Signed-off-by: Xiaodai Wang <xiaodwan@redhat.com>

### CODE REVIEW

**Comment by chunfuwen** [comment]
Any test result available?

**Comment by xiaodwan** [comment]
2020-06-28 05:54:28,527 test             L1018 INFO | PASS 1-type_specific.io-github-autotest-libvirt.function_test_esx.positive_test.windows.OGAC.win10.esx_67.rhev.rhv_upload.it_vddk.vpx_uri

**Comment by xiaodwan** [comment]
> Any test result available?

**Comment by xiaodwan** [comment]
@chunfuwen @chloerh @yantian0 Please help to review this PR, thanks.

File: virttest/utils_v2v.py [hunk 1]
```diff
@@ -455,6 +455,9 @@ def __init__(self, test, params, env):
             raise ValueError("Doesn't support %s target now" % self.target)
 
     def create_session(self, timeout=480):
+        if self.session:
```
Comment (chloerh) [inline virttest/utils_v2v.py:458]: Are we considering adding another parameter to have options on this? Like, 'single_session=True'

Comment (xiaodwan) [reply virttest/utils_v2v.py:458]: Right now we don't have such use case.  It could be implemented if needed in future.

**Pull request merged**